content/articles/binance-smart-chain-tutorial/index.md

```diff
@@ -0,0 +1,241 @@
+
+### Why Binance Smart Chain?
+Over time the Ethereum platform has expanded causing network congestion to its infrastructure. This impacted gas fees and slow transactions. As the plan to migrate to Proof-of-Stake has taken time, Ethereum currently uses Proof-of-Work consensus. On the other hand, Binance uses the Proof-of-Authority mechanism.
+
+Block producers are known as validators in the Proof-of-Authority consensus in Binance Smart Chain. These validators are limited to 21 and need the approval of identities from Binance. Since we need pre-approval from Binance to become a validator, Binance gains complete control making it more of a private blockchain. The Binance Smart Chain is fully compatible with the Ethereum Virtual Machine.
+
+### Set Up Remix
+Remix is an online tool for Ethereum and solidity development. It comes bundled with plugins for testing, debugging, and deploying smart contracts. Launch Remix IDE on your browser by navigating to https://remix.ethereum.org/. The editor looks like:
+
+![Remix IDE](/engineering-education/binance-smart-chain-tutorial/remix.png)
+
+In your environment, select the solidity version to use. In our case, `version 0.8.2`. Under the `workspaces`, inside our `contracts` folder, add a new file and name it `Token.sol`. Every Solidity file requires that we add the license type and the solidity version compatible with the code and configurations. Inside your `Token.sol`, add the two lines of code as:
+
+```js
+//  SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2; // compiler version
+```
+The above code is very straightforward:
+
+- Solidity will use [SPDX](https://spdx.dev/) that has been specified on the first line as `SPDX-License-Identifier: MIT` to show that our code is compliant with the MIT license. 
+- The line statement `pragma solidity ^0.8.2;` tells solidity that our code is compatible with version `0.8.2` or later. 
+
+> Note: A pragma directive remains local to a source file ensuring that any modules we import on a file do not cause ambiguity.
+
+### Tokens in Binance Smart Chain
+In blockchains that support smart contracts, developers need to create crypto tokens that are based on a set of standards. This includes building DApps, exchanges, wallets, etc. While Ethereum's popular standard is the ERC-20 token, BSC uses the BEP-20 standard and in many ways, it is similar to the ERC-20 in Ethereum. Examples of popular BEP-20 tokens include:
+
+- BUSD Token: BUSD stands for [Binance USD](https://paxos.com/busd/). It is a stable coin created by [Paxos](https://paxos.com/), a tether to USD.  
+- CAKE Token: This is a BEP-20 token that controls the [PancakeSwap](https://pancakeswap.finance/) platform.
+- BUX Token was created as a utility token for the [BUX exchange](https://bux-c.com/) ecosystem.
+- SAFEMOON Token: a Defi token for [safemoon](https://safemoon.net/) on the Binance Smart Chain.
+
+### Defining Contracts and Constructor
```

Comment:
You have defined a contract but not a constructor

Reply:
should be Contract alone since smart contract constructor is below :hand: